Pr add domd image weston

andr2000 opened the pull request.

Commit:
domd: Add core-image-weston for driver domain

Currently we can only build core-image-weston for Dom0 or DomU,
so make it possible to build the same for driver domain.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>

lorc commented on recipes-domd/domd-image-weston/domd-image-weston.bb:
```diff
@@ -0,0 +1,14 @@
+
+LICENSE = "MIT"
+
+require recipes-dom0/dom0-image-weston/dom0-image-weston.bb
```
I don't think, that we should rely on dom0 receipt. Who knows, what will change in dom0 recipe in the future. 
Also, it is rather strange to include .bb file into another .bb file.

I would recommend to move all shared code into separate .inc file and then include it in Dom0, DomU and DomD recipes.

Commit:
domx: Use generic part for core-image-weston in recipes

Dom0 and DomU recipes domx-image-weston are almost the same,
so move common code into include file and use it.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>

lorc commented:
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`
